<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>460</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>11111</width>
    <height>11111</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Selection History</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>11111</width>
     <height>11111</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>History</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Selection History:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="selection_history_list">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>222</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="midLineWidth">
             <number>0</number>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="resizeMode">
             <enum>QListView::Adjust</enum>
            </property>
            <property name="layoutMode">
             <enum>QListView::SinglePass</enum>
            </property>
            <property name="uniformItemSizes">
             <bool>false</bool>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clear_history_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Clear History</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Saved</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Saved Selections:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="saved_selections_list">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>222</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="save_button">
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="text">
             <string>Save Selection</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="clear_saved_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Clear Saved</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="import_button">
            <property name="text">
             <string>Import..</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="export_button">
            <property name="text">
             <string>Export..</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>160</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Max History:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="max_history">
             <property name="maximum">
              <number>200</number>
             </property>
             <property name="singleStep">
              <number>5</number>
             </property>
             <property name="value">
              <number>20</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="double_click_checkbox">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Double Click to Select</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="include_single_checkbox">
           <property name="text">
            <string>Include Single Selections</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
